src/anbox/cmds/check_features.cpp

```diff
@@ -30,22 +30,22 @@ std::vector<std::string> cpu_whitelist = {
   // The following CPUs do not support AVX and without it cpu_features can't detect
   // if SSE & friends are supported. See https://github.com/google/cpu_features/issues/4
 
+  // Intel Core i7 Q720
+  "Q 720",
   // Intel Core i7 M620
   "M 620",
   // Intel Core i5 M460
   "M 460",
   // Intel Celeron N2840
   "N2840",
-  // Intel Core i7 Q720
-  "Q 720",
-  // Intel Pentium T4500
-  "T4500", 
-  // Intel Core i7 Q720
-  "Q 720",
   // Intel Xeon E5520
-  "E5520"
+  "E5520",
+  // Intel Pentium T4500
+  "T4500",
   // Intel Core2 Duo T6500
-  "T6500"
+  "T6500",
+  // Intel Core2 Duo T9600
+  "T9600"
```

Comment:
According to http://www.cpu-world.com/CPUs/Core_2/Intel-Core%202%20Duo%20Mobile%20T9600%20AW80576GH0726M%20(BX80576T9600).html the CPU doesn't support SSE 4.2 so we can't add it here.